Exercise_8/Resources/values/attrs.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<resources>
+  <declare-styleable name="SetProgressProperties">
```

Comment:
Bad name, it can be `progress` (single noun and lower case, pascalStyle)

Reply:
Done